forest/cli/src/cli/mod.rs

```diff
@@ -187,6 +187,7 @@ pub(super) fn print_stdout(out: String) {
         .unwrap();
 }
 
+// TODO(aatifsyed): replace with dialoguer
```

Comment:
Let's have an issue with this - TODOs without a linked issue are expected to be resolved within the same PR. At least, that's my practice. 

Also, as far as I remember, we re-implemented it here for this reason: https://github.com/console-rs/dialoguer/issues/170

But it would be nice not to use such workarounds.

Reply:
https://github.com/ChainSafe/forest/issues/2928